ST_mortality.c

```diff
@@ -68,70 +71,74 @@ void _kill_maxage(void);
 
 /************ File-Level Variable Declarations *************/
 /***********************************************************/
+/**
+ * \brief TRUE if a function in ST_mortality.c killed an individual.
+ * \ingroup MORTALITY_PRIVATE
+ */
 Bool _SomeKillage;
-/* flag: some plant was reduced and PR is affected. */
-/* 7/5/01  - currently flag is set but unused. */
+
 extern
   pcg32_random_t mortality_rng; //declared in ST_main.c
 
-
-/***********************************************************/
-/***********************************************************/
+/**
+ * \brief Performs mortality that occurs during the growing season.
+ * 
+ *  This function reduces "plant-space" according to
+ *  resource availability, age, growth rate, and disturbance
```

Comment:
replace , and disturbance with:

, and fecal pats, ant mounds, and animal burrows

Reply:
Addressed by commit 984bd13.